encryption_formats.md

```diff
@@ -90,7 +90,7 @@ None (but don’t use the same key more than once).
 
 #### Usage
 
-This format is used if the customer disabled the padding option. Otherwise, the v6 format is used.
+This format was instead of v6 when the padding option was disabled. It is no longer used.
```

Comment:
used?

Reply:
```suggestion
This format was used instead of v6 when the padding option was disabled. It is no longer used.
```

Reply:
Still, the wording seems unclear to me.

Maybe we should reword like: `This format is no longer used. It has been superseded by ...`. What do you think?

Reply:
Updated this one and v6 with "superseded" wording, let me know if the wording looks reasonable

Reply:
LGTM